core/trino-main/src/main/java/io/trino/operator/output/PositionsAppenderFactory.java

```diff
@@ -18,12 +18,29 @@
 import io.trino.spi.type.FixedWidthType;
```

Comment:
Improve commit description

Reply:
Not sure what you mean. PTAL if now it is bettter